analyses/pypi/pypi.go

```diff
@@ -0,0 +1,107 @@
+package pypi
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/rs/zerolog/log"
+)
+
+// GeneratePylist creates pylist.json from requirements.txt
+func GeneratePylist(shellPath string, manifestFilePath string) string {
```

Comment:
Don't deal with primitives, there are types to represent path(`path/filepath`).

Reply:
there is no type as `Path` or `filepath`. it's all `string` which it takes and outputs